rbac/migration_tool/utils.py

```diff
@@ -0,0 +1,30 @@
+"""Utilities for working with the relation API server."""
+import grpc
+from django.conf import settings
+from relations.v0 import common_pb2
+from relations.v0 import relation_tuples_pb2
+from relations.v0 import relation_tuples_pb2_grpc
+
+
+def create_relationship(resource_name, resource_id, subject_name, subject_id, relation):
+    """Create a relationship between a resource and a subject."""
+    return common_pb2.Relationship(
+        resource=common_pb2.ObjectReference(type=common_pb2.ObjectType(name=resource_name), id=resource_id),
+        relation=relation,
+        subject=common_pb2.SubjectReference(
+            subject=common_pb2.ObjectReference(type=common_pb2.ObjectType(name=subject_name), id=subject_id)
+        ),
+    )
+
+
+def write_relationships(relationships):
+    """Write relationships to the relation API server."""
+    with grpc.insecure_channel(settings.RELATION_API_SERVER) as channel:
+        stub = relation_tuples_pb2_grpc.KesselTupleServiceStub(channel)
+
+        request = relation_tuples_pb2.CreateTuplesRequest(
+            upsert=True,
+            tuples=relationships,
+        )
+
+    return stub.CreateTuples(request)
```

Comment:
I believe we can add some try catch block to catch errors.

@vbelchio can you show example how do you handle errors  for gRPC requests ?
We were discussing and I can't find link to that discuss.
I would prefer to see error message from gRPC which in which attribute proper error message is, ....
(With that I will also add such examples in repo client)

Reply:
Updated